test/targets.js

```diff
@@ -77,10 +77,13 @@ var itShouldGenerateOutput = function (request, path, target, client) {
       this.skip()
     }
     var instance = new HTTPSnippet(fixtures.requests[request])
-    var result = instance.convert(target, client) + '\n'
+
+    // `form-data` sets the line break as `\r\n`, but we can't easily replicate that in our fixtures so let's convert
+    // it to a standard line break instead.
+    var result = instance.convert(target, client).replace(/\r\n/g, '\n').trim()
```

Comment:
👍 

I'm guessing this is the cause of all the whitespace differences in many of the fixture output files?

Reply:
Yeah that, and a lot of those files had `CRLF` line endings instead of `LF` (which is what's configured in `.editorconfig`.